instance/models/mixins/ansible.py

```diff
@@ -54,6 +54,8 @@ class AnsibleAppServerMixin(models.Model):
         help_text='YAML variables for commonly needed services.')
 
     INVENTORY_GROUP = 'generic'
+    # It is possible to get an IP address that has already appeared, so we want to disable strict host key checking.
```

Comment:
Drive by comment: can this comment include a warning that this is a special case and disabling strict host key checking is not something one should do?